paper.tex

```diff
@@ -427,16 +427,22 @@ \subsection{Classes}
 It can be mapped to the following EO code, where a class becomes a factory of objects:
 
 \begin{ffcode}
++alias org.eolang.cage
++alias org.eolang.txt.sprintf
++alias org.eolang.io.stdout
++alias org.eolang.hamcrest.assert-that
```

Comment:
@ivan-egorov42 I don't see any `assert-that` objets in this code. Do really we need it here?

Reply:
My bad, I'll remove this.